gen-app-builder/search-web-app/templates/index.html

```diff
@@ -74,6 +74,7 @@ <h1>Enterprise Search Demo - Widgets</h1>
         triggerId="{{s.name|lower|replace(' ', '-')}}"
       >
       </gen-search-widget>
+      <div class="search-engine-name"><b>{{s.name}}</b></div>
```

Comment:
The UX here is a bit jarring when clicking on one of the text search boxes out of the many, even for me who is familiar with how that ES widget works as a full-screen popover.

Perhaps add a note at the top of the page that clicking on one of the text boxes will open up a full-page search widget for that particular engine? Or maybe a different UX element would help here such as tabs for each search engine?

Reply:
I would agree that a different design would make more sense. TBH, this whole app design could probably be redone to make it more "reactive", but I just tried to get it out quickly :P

I'll add a section at the top clarifying the behavior